src/generic/grid.cpp

```diff
@@ -11337,4 +11400,640 @@ wxGetContentRect(wxSize contentSize,
     return contentRect;
 }
 
+
+#if wxUSE_ACCESSIBILITY
+
+//-----------------------------------------------------------------------------
+// helpers for wxGridAccessible and wxGridCellAccessible
+//-----------------------------------------------------------------------------
+
+// helper to convert row / col into child object id
+// error handling is not required here
+static int GetChildId(wxGrid* grid, int row, int col,
```

Comment:
It might be worth renaming this to `ConvertRowColToChildId()` and the function below to `ConvertChildIdToRowCol()` to emphasize that they're symmetric to each other.